src/omotes_sdk/internal/worker/params_dict.py

```diff
@@ -46,11 +80,16 @@ def parse_workflow_config_parameter(
     of_type = type(maybe_value)
     is_present = field_key in workflow_config
 
+    result: ParamsDictValue
     if is_present and isinstance(maybe_value, expected_type):
         # cast is necessary here as Expected type var may not have the same type as
         # `workflow_config[field_key]` according to the type checker. However, we have confirmed
         # the type already so we may cast it.
         result = cast(ParamsDictValue, maybe_value)
+    elif is_present and type(maybe_value) is float and expected_type is datetime:
+        result = cast(ParamsDictValue, datetime.fromtimestamp(maybe_value))
```

Comment:
Just to make sure: no try catch needed because we only expect to parse parameters which were correctly created in `convert_params_dict_to_struct`?

Reply:
The `cast()` is only for the typechecker. During runtime it has no function. As maybe_value is a float, `datetime.fromtimestamp` should always succeed. The rest of the `elif` tree should take care of all erronuous/other cases so I do not expect a try/catch necessary on this specific line. Does this answer your Q?

Reply:
yes, tnx